modules/reactor/src/meshgenerators/AzimuthalBlockSplitGenerator.C

```diff
@@ -90,6 +90,13 @@ AzimuthalBlockSplitGenerator::generate()
       paramError("old_blocks",
                  "This parameter contains blocks that do not exist in the input mesh.");
 
+  if (std::find(_old_block_ids.begin(), _old_block_ids.end(), mesh.elem_ref(0).subdomain_id()) !=
+          _old_block_ids.end() &&
+      mesh.elem_ref(0).n_vertices() == 4)
```

Comment:
checking the number of vertices of element 0 might not be an accurate way of checking for center quad elements?

I think I d rather use the meta data here

Reply:
Sure